core/controllers/base.py

```diff
@@ -41,3 +43,20 @@ def render_json(self, values):
         json_output = json.dumps(
             values, cls=custom_json_encoder.JSONEncoderForHTML)
         self.response.write('%s%s' % (config.XSSI_PREFIX, json_output))
+
+    def render_templates(self, filepath):
```

Comment:
Just checking, why do we need this? Shouldn't all static pages be served directly from the static handler?

Prefer not to have this if possible, I think. Can we find a way to remove it?

Reply:
You know, I was too occupied thinking that I needed a template rendering without really checking whether things would work without these back-end additions.

Yes, things still work with just static handler. I will remove these codes.